packages/terra-alert/package.json

```diff
@@ -29,9 +29,7 @@
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
     "terra-button": "^3.29.0",
-    "terra-doc-template": "^2.23.0",
     "terra-icon": "^3.25.0",
-    "terra-markdown": "^2.37.0",
```

Comment:
you deleted the Markdown dep, but then its being used in the About mdx file: https://github.com/cerner/terra-core/pull/2813/files?short_path=62ce338#diff-5c324b96e5cac7b2608bf3e696e02325R73

Reply:
removed it here https://github.com/cerner/terra-core/pull/2813/commits/0ec3e9b2b11243ff5a0c25908ee356a9de91d5a8.